### DISCUSSION TITLE
patch file size after domain_substitution?

### DISCUSSION THREAD

**Discussion by cjbd** [Eloston/ungoogled-chromium#1403] [Q&A]
hello, i'm using this project(with minor update) for my linux local chromium source cloned using git and gclient tools,
i've come to `domain_substitution.pu` step, after that, i generated a patch file using
```
git diff > mypatch.patch
```
the file is huge, for me it's 114.3MB, even after applying this PR#1026, is this normal?

**Comment by wchen342** [comment]
That seems too large..the Bromite one is like 2.5MB. Check whether you got some binary files inside.

**Reply by cjbd** [reply]
i'm uploading both domain_substitution.list and chromium.patch files, seems the first list file is bigger than the one comes with this project

[chromium.zip](https://github.com/Eloston/ungoogled-chromium/files/6017154/chromium.zip)
[domain_substitution.zip](https://github.com/Eloston/ungoogled-chromium/files/6017152/domain_substitution.zip)
